test-cypress/integration/ttyg/chat-list.spec.js

```diff
@@ -129,9 +141,40 @@ describe.skip('TTYG chat list', () => {
         TTYGViewSteps.getChatByDayGroups().should('have.length', 1);
     });
 
+    // FIXME: Skipped because of exception thrown by GDB
```

Comment:
What is the exception which GDB throws? For the TTYG tests we must mock all GDB API calls because it relies on an external service (the OpenAI API).

Reply:
Actually I got a bit confused.
I will remove the todos and enable the tests.

I have mocked them and they pass with the mocks.

The exception is due to a problem in how the response is sent by GraphDB, which is reported and will be fixed.